CONTRIBUTING.md

```diff
@@ -145,6 +146,11 @@ via email is also possible - contact maintainer about the details.
 Code submitted as raw source files (not patches), files attached to issue
 comments, forum posts, diffs in non-git format, etc will be promptly ignored.
 
+Any changes to README.md have to consider the project's latest stable release, 
+not the latest commit from the alpha version. New feature additions and removals
```

Comment:
Hmm, I don't agree with this… README.md file is for developers - the primary target group are people compiling the `main` branch.

That being said, I understand why this paragraph was added, perhaps on the top of README.md we should put a link to README.md for the latest stable release?

Reply:
Yeah I was 50/50 but then Thomas made me unsure. I'd prefer README.md to reflect the current state of main too. Then the front page of the website will list the main highlights of the current release.

Reply:
@johnnovak I agree with @dreamer here. Most parts of the README.md should be for the developers. However, I think the `Summary of Features` section specifically should reflect the state of the latest stable release. That's at least how @kcgen has been doing it, changing it as a part of the release steps. This added section to the `CONTRIBUTING.md` is all about preventing the change request https://github.com/dosbox-staging/dosbox-staging/pull/2456#pullrequestreview-1421843408 from happening and trying to standardize a set of rules for changing readme.md. 

I've sent the context above (the change request) to @dreamer and asked him if he could wager on the situation. And he has, see: https://github.com/dosbox-staging/dosbox-staging/pull/2456#discussion_r1191258346 . He suggests we should remove "summary of features" table, in which I agree.

Reply:
See my last comment and @kcgen's comments in the linked thread. This table will stay until the manual is out.